### DISCUSSION TITLE
Error on deploying from script

### DISCUSSION THREAD

**Discussion by Rayyan-Uddin** [Cyfrin/foundry-full-course-cu#2481] [Q&A] [answered]
I'm having this error while deploying contract from script command. Please guide.

![error](https://github.com/user-attachments/assets/ee08f961-c2cb-452e-86bb-19079ff31f23)

**Comment by EngrPips** [comment, marked as answer]
Hello @Rayyan-Uddin, Please run `forge clean` and then read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2474 . It should help you resolve this issue.

**Reply by Rayyan-Uddin** [reply]
Thankyou..

**Reply by EngrPips** [reply]
You are welcome.
